pkg/counter/cache.go

```diff
@@ -16,14 +17,20 @@ var log = logf.Log.WithName("counter_cache")
 
 var cachedCounts = cache{
 	Counts: Counts{
+		MasterUserRecordCount:        0,
 		UserAccountsPerClusterCounts: map[string]int{},
+		UsersPerActivationCounts:     map[int]int{},
 	},
 }
 
 // Counts is type that contains number of MURs and number of UserAccounts per member cluster
 type Counts struct {
-	MasterUserRecordCount        int
+	// Total number of Master User Accounts
+	MasterUserRecordCount int
+	// User Accounts per Clusters (indexed by cluster name)
 	UserAccountsPerClusterCounts map[string]int
+	// Activations per Users (number of users indexed by their number of activations)
```

Comment:
It's not activations per user. It's number of users per activation count. Correct?

Reply:
ah right, I cherry-picked a change to make a separate PR and:
- I did not realize this file was affected in this PR (it should not)
- I forgot to edit the commend (the initial var name was misleading, as you noted)

Reply:
removed in f9a2b08